library/src/main/kotlin/com/a99/rxplaces/GeocodeRepositoryImpl.kt

```diff
@@ -0,0 +1,31 @@
+package com.a99.rxplaces
+
+import com.a99.rxplaces.GoogleMapsApi.Companion.INVALID_REQUEST
+import com.a99.rxplaces.GoogleMapsApi.Companion.OK
+import com.a99.rxplaces.GoogleMapsApi.Companion.OVER_QUERY_LIMIT
+import com.a99.rxplaces.GoogleMapsApi.Companion.REQUEST_DENIED
+import com.a99.rxplaces.GoogleMapsApi.Companion.UNKNOWN_ERROR
+import com.a99.rxplaces.GoogleMapsApi.Companion.ZERO_RESULTS
+import rx.Observable
+
+
+internal class GeocodeRepositoryImpl (val api: GoogleMapsApi, val key: String) : GeocodeRepository {
+  override fun reverseGeocode(placeId: String): Observable<GeocodeResult> {
```

Comment:
Yeah, sure. Once It was already merged, feel free to create a PR with this change.